app/mailers/password_reset_mailer.rb

```diff
@@ -1,11 +1,11 @@
 class PasswordResetMailer < ActionMailer::Base
   helper :application
   include ApplicationHelper
-  default from: "notifications@#{ActionMailer::Base.default_url_options[:host]}"
+  default from: "Public Lab <notifications@#{ActionMailer::Base.default_url_options[:host]}>"
 
   # PasswordResetMailer.reset_notify(user).deliver_now
   def reset_notify(user, key)
-    subject = '[Public Lab] Reset your password'
```

Comment:
why are you removing this?

Reply:
Because there are two tasks given in this issue. Given by @SidharthBansal

Reply:
According to me it will not be necessary if we show public lab instead of notification

Reply:
I am saying this why we need to remove this ?
@SidharthBansal why we need to remove this?

Reply:
If it is showing the sender name is Public lab then no requirement of [Public Lab] agian in the title

Reply:
done

Reply:
Here I think this is OK! See how things appear in Gmail - i think Public Lab will appear to the left:

![image](https://user-images.githubusercontent.com/24359/45912965-27149700-bdf8-11e8-9c77-7796352ed412.png)